File: app/src/main/java/com/pkmnapps/activitydo/dataclasses/ActivityData.java
package com.pkmnapps.activitydo.dataclasses;

import android.graphics.Bitmap;
import android.graphics.Color;

import java.util.Date;

public class ActivityData {
    private String name,id;
    private String color;
    private Boolean pinned;
    private int sortOrder;

    public ActivityData() {
    }

    public ActivityData(String id, String name, String color) {
        this.name = name;
        this.id = id;
        this.color = color;
        this.pinned = false;
    }

    public ActivityData(String id, String name, String color, Boolean pinned, int sortOrder ) {
        this.name = name;
        this.id = id;
        this.color = color;
        this.pinned = pinned;
        this.sortOrder = sortOrder;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getColor() {
        return color;
    }

    public void setColor(String color) {
        this.color = color;
    }

    public Boolean getPinned() {
        return pinned;
    }

    public void setPinned(Boolean pinned) {
        this.pinned = pinned;
    }

    public int getSortOrder() {
        return sortOrder;
    }

    public void setSortOrder(int sortOrder) {
        this.sortOrder = sortOrder;
    }
}

File: app/src/main/java/com/pkmnapps/activitydo/dataclasses/ActivityData.kt
package com.pkmnapps.activitydo.dataclasses

import org.junit.runner.RunWith

class ActivityData {
    private var name: String? = null
    private var id: String? = null
    private var color: String? = null
    private var pinned: Boolean? = null
    private var sortOrder = 0

    constructor() {}
    constructor(id: String?, name: String?, color: String?) {
        this.name = name
        this.id = id
        this.color = color
        pinned = false
    }

    constructor(id: String?, name: String?, color: String?, pinned: Boolean?, sortOrder: Int) {
        this.name = name
        this.id = id
        this.color = color
        this.pinned = pinned
        this.sortOrder = sortOrder
    }

    fun getId(): String? {
        return id
    }

    fun setId(id: String?) {
        this.id = id
    }

    fun getName(): String? {
        return name
    }

    fun setName(name: String?) {
        this.name = name
    }

    fun getColor(): String? {
        return color
    }

    fun setColor(color: String?) {
        this.color = color
    }

    fun getPinned(): Boolean? {
        return pinned
    }

    fun setPinned(pinned: Boolean?) {
        this.pinned = pinned
    }

    fun getSortOrder(): Int {
        return sortOrder
    }

    fun setSortOrder(sortOrder: Int) {
        this.sortOrder = sortOrder
    }
}